Dockerfile

```diff
@@ -15,4 +15,8 @@ RUN systemctl set-default multi-user.target
 
 COPY setup /sbin/
 
-CMD ["/sbin/init"]
+STOPSIGNAL SIGRTMIN+3
+VOLUME ["/sys/fs/cgroup"]
+
+# Workaround for docker/docker#27202
+CMD ["/bin/bash", "-c", "exec /sbin/init --log-target=journal 100>&1 1>&-"]
```

Comment:
Could you explain what's going on here? If I'm reading this correctly, you're redirecting file descriptor 100 to `stdout` and then closing `stdout`. What's the significance of FD 100 for systemd, and why does `stdout` need to be closed?